File: LV0/배열 만들기 6.js
function solution(arr) {
  let answer = [];
  for (let i = 0; i < arr.length; i++) {
    if (answer.length === 0) {
      answer.push(arr[i]);
    } else if (answer.length > 0) {
      if (answer[answer.length - 1] === arr[i]) {
        answer.pop();
      } else {
        answer.push(arr[i]);
      }
    }
  }
  return answer.length > 0 ? answer : [-1];
}

File: LV0/배열 만들기 6.py
def solution(arr):
    answer = []
    for i in range(len(arr)):
        if len(answer) == 0:
            answer.append(arr[i])
        elif len(answer) > 0:
            if answer[-1] == arr[i]:
                answer.pop()
            else:
                answer.append(arr[i])
    return answer if len(answer) > 0 else [-1]
